modules/ROOT/pages/amazon/amazon-ec2-connector.adoc

```diff
@@ -401,17 +557,17 @@ image::amazon/amazon-ec2-create-volume-payload.png[ec2 create volume payload tra
 +
 . Add a Logger component after the Set Payload transformer to print the Volume ID that is being transformed by the Set Payload transformer from the Create Volume operation in the Mule Console. Configure the Logger according to the table below.
 +
-[%header%autowidth.spread]
+[%header,cols="30s,70a"]
 |===
-|Parameter |Value
+|Studio Field |Value
 |Display Name |Logger (or any other name you prefer)
 |Message |`#[payload]`
 |Level |INFO
 |===
 +
 image::amazon/amazon-ec2-create-volume-logger-props.png[ec2 create volume logger]
 +
-. Save and Run the project as a Mule Application. Right-click the project in Package Explorer and click Run As > Mule Application.
+. Save and Run the project as a Mule Application. Right-click the project in Package Explorer and click *Run As* > *Mule Application*.
 . Open a browser and check the response after entering the URL `+http://localhost:8081/createVolume+`. You should see the generated Volume ID in the browser and the console.
 
 [[example-code]]
```

Comment:
Line 567
Paste this code into your XML editor to quickly load the flow for this example use case to your Mule app: